app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -222,6 +222,72 @@ public void onButtonSwitchToUserProfileActivity(View view) {
         setPutExtraArguments(playerProfileIntent);
         startActivity(playerProfileIntent);
     }
+
+
+    public void onButtonShowQuestionPopupWindowClick(View view, Boolean focusable, int layoutId, Riddle riddle) {
```

Comment:
This is a bad merge from master, it should not be here

Reply:
removed